From: Michael Tokarev <mjt@tls.msk.ru>
Date: Wed, 14 Feb 2024 15:26:29 +0300
Subject: silence "Can not convert group sid" warnings in the log
Forwarded: yes

/var/log/samba/log.winbind is full of messages like:

[2024/02/09 06:25:04.788182,  1, pid=74620] source3/winbindd/winbindd_getgroups.c:259(winbindd_getgroups_recv)
  Could not convert sid S-0-0: NT_STATUS_NONE_MAPPED

On a busy server these are logged several 1000s times per minute,
making any other messages basically invisible.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>

diff --git a/source3/winbindd/winbindd_getgroups.c b/source3/winbindd/winbindd_getgroups.c
index c2603cc7026..f252e14bf95 100644
--- a/source3/winbindd/winbindd_getgroups.c
+++ b/source3/winbindd/winbindd_getgroups.c
@@ -257,5 +257,5 @@ NTSTATUS winbindd_getgroups_recv(struct tevent_req *req,
 	if (tevent_req_is_nterror(req, &status)) {
 		struct dom_sid_buf buf;
-		D_WARNING("Could not convert sid %s: %s\n",
+		D_DEBUG("Could not convert sid %s: %s\n",
 			  dom_sid_str_buf(&state->sid, &buf),
 			  nt_errstr(status));
